### DISCUSSION TITLE
How to replace node-sass?

### DISCUSSION THREAD

**Discussion by markkra** [parcel-bundler/parcel#5326] [Q&A] [answered]
Our company has a fancy node-sass wrapper that does some token resolving for various brands before calling node-sass. Could I cut in with a custom Parcel plugin that would say "when you go to use node-sass, use our uds-node-sass instead"?

**Comment by mischnic** [comment, marked as answer]
This should be possible using your package manager, without changing Parcel:

```json
"dependencies": {
  "node-sass": "npm:uds-node-sass@^5.0.0"
}
```

**Reply by markkra** [reply]
Way cool! I'll give it try. Way better than grabbing https://github.com/parcel-bundler/parcel/blob/52f365668013a08b8afa03e549934a6137eb9c4d/packages/core/parcel-bundler/src/assets/SASSAsset.js and tweaking getSassRuntime() and adding an asset plugin.

**Comment by markkra** [comment]
That worked! I have other dependency issues but it's using the correct compiler
